src/pages/household/output/PolicyImpactPopup.jsx

```diff
@@ -11,7 +11,15 @@ export default function PolicyImpactPopup(props) {
     setHasShownPopulationImpactPopup,
   } = props;
   const content = (
-    <div>
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        textAlign: "center",
```

Comment:
```suggestion
        textAlign: "center",
        gap: "8px",
```

Reply:
This and the one below are because in the original, some margins collapse together, but in yours, they don't